src/DBFTPlugin/DBFTPlugin/config.json

```diff
@@ -5,6 +5,6 @@
     "AutoStart": false,
     "Network": 860833102,
     "MaxBlockSize": 2097152,
-    "MaxBlockSystemFee": 100000000000
+    "MaxBlockSystemFee": 300000000000
```

Comment:
1500 or 2000 GAS? 3K GAS is still quite a lot of CPU time.

Reply:
> 1500 or 2000 GAS? 3K GAS is still quite a lot of CPU time.

Done.